### DISCUSSION TITLE
Problem with Dexcom g6 bluetooth connection

### DISCUSSION THREAD

**Discussion by bensquires** [NightscoutFoundation/xDrip#3452] [Bug reports]
I use Samsung s21 ultra with G6 transmitter and latest version of xdrip. It has been working fine for years.
In the last couple of weeks, it has started regularly loosing blutooth connection. Especially a problem at night, as the alarm wakes my wife and a I everytime it tries to reconnect. Any ideas?

**Comment by Navid200** [comment]
Please post a screenshot of the status page.

**Reply by bensquires** [reply]
![Screenshot_20240510_140203_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/8eed1f31-f448-4793-8d21-ee2bef09dcb9)
![Screenshot_20240510_140155_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/e3e1f5a1-2730-42c4-8c12-9d2280de49b6)

**Reply by Navid200** [reply]
This is Dexcom One, not G6.

Please post a screenshot of the following page:

Settings &#8722;> G5/G6/G7/Dex1 Debug Settings

**Reply by bensquires** [reply]
Thanks for for your help. I should have made it clear that it was Dexcom ONE rather than G6 but they seem to have identical hardware and I was trying to avoid confusion. I will attach the screenshot. Thanks again Ben 🙂
![Screenshot_20240510_154938_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/cd7c322f-f74c-401f-8410-961d2bf7a530)
![Screenshot_20240510_154953_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/4a481bc2-4476-473b-9be7-a10324432eba)

**Reply by Navid200** [reply]
You have not shown all the settings.  But, the ones you have shown are incorrect.

Follow this guide: https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

After you use the recommended settings, if the problem continues, let us know.

**Reply by bensquires** [reply]
Thanks so much for your help. Much appreciated! 👏👏👏

**Reply by bensquires** [reply]
So far, so good.
Thanks again.
I am amazingly grateful for the efforts of people like yourself who support people like me to use fantastic resources like xdrip. Thanks again. Ben

**Reply by Navid200** [reply]
You're welcome
Thanks

**Comment by bensquires** [comment]
Hi, sorry to bother again, but I am still having problems with blutooth connection to the Dexcom transmitter. It looses contact several times a day and at night, triggering a pairing request on my phone.
I have double checked the xDrip settings as requested. 
I have checked the battery settings for xdrip and blutooth app and blutooth agent.
I have checked the other permission settings.
I have worked through the recommended steps on the github document. 
Any ideas?
Attached is a log.
Many thanks,
Ben
![Uploading Screenshot_20240522_075151_xDrip+.jpg…]()
![Uploading Screenshot_20240522_075111_xDrip+.jpg…]()

**Comment by bensquires** [comment]
Not sure my screenshot has uploaded. Will try again.
![Screenshot_20240522_075151_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/cddb47df-fb01-4d6e-8224-8fc7bc055b7e)
![Screenshot_20240522_075111_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/58544304-f954-4152-a7d3-7acefefc4bbf)

**Reply by Navid200** [reply]
These are multiple questions and requests:

Post a screenshot of the G5/G6/G7 Status page.
Are you sure you have disabled "Allow OB1 unbonding"?
Are you sure you have enabled "Special pairing workaround"?

**Comment by bensquires** [comment]
Many thanks,
Here is a screen shot of the status page
![Screenshot_20240522_192216_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/44000931/42c288d7-9a7b-4471-ba9d-91f01408edcb)
"Allow OB1 unbonding" is disabled.
"Special pairing workaround" was NOT enabled. I have enabled it now. Hopefully that might work???
Many thank again for your expertise.
Kind regards. Ben

**Comment by bensquires** [comment]
Thanks again for your help. 
Your suggestion of enabling the "special pairing workaround" seems to have fixed the problem.
I am enormously grateful for your input.
xDrip is an amazing resourse and thank you for  your support. 👍

**Reply by Navid200** [reply]
You're welcome
